### PR TITLE
#184637119 Set up an empty Express Boilerplate with dotenv

### PR DESCRIPTION
**Pull request by mjeanbosco19**
### What does this PR do?

- Set up an empty Express Boilerplate with dotenv.

### Description of Task to be completed?

- Set up an empty Node.js - express application.  Full directory with all folders that you think developer will need

### How should this be manually tested?

**Installation**
**1. Clone the repository by running git clone https://github.com/atlp-rwanda/destructors-ec-bn.git in your terminal.**
```
https://github.com/atlp-rwanda/destructors-ec-bn.git
```
**2. Navigate to the project directory by running cd destructors-ec-bn.**
```
cd destructors-ec-bn
```
**3. Install the necessary dependencies by running npm install.**
```
npm install
```
**4. Run the development server by running npm run dev. This will start the server using Nodemon, which will automatically reload the server when changes are made to the code.**
```
npm run dev
```

### Any background context you want to provide?

- none

### What are the relevant pivotal tracker/Trello stories?

- Tracker: 184637119

### Screenshots (if appropriate)

- none

### Questions:

- none

### CODE REVIEW

**Review by istafrica-dennw — CHANGES_REQUESTED** [review]
rename the commit mesage by removing the task id,

**Comment by mjeanbosco19** [comment]
> rename the commit mesage by removing the task id,

Thanks, I have completed to rename the commit message.

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import app from './app.js';
+import dotenv from 'dotenv';
+
+dotenv.config();
```
Comment (Abdulkeza) [inline src/index.js]: You can change this into  a single line by using `import 'dotenv/config'`

**Review by Abdulkeza — CHANGES_REQUESTED** [review]
Nice Job @mjeanbosco19 , but I can't see all folders as required.
Also in PR body under **How should this be manually tested?** it should be better if we add the link to clone
ex: _git clone [repo](https://github.com/atlp-rwanda/git-practices-destructors.git)_
     _cd destructors-ec-bn_
     _npm install_
    _npm run dev_
     _etc_     coz I know you can't run the project without installing dependencies.

File: src/index.js (hunk 1 above)
Comment (leandreAlly) [reply src/index.js]: @Abdulkeza  Thanks for this pro tips!

**Review by Abdulkeza — APPROVED** [review]
Nice work @mjeanbosco19 .

**Review by katros1 — APPROVED** [review]

**Review by divinecharlotte — APPROVED** [review]
# Hello @mjeanbosco19    😃 🥳 🎉 🎈 

Great Job 😄 👏 You've done it. There is nothing else to say other than... it's time to merge it 👍🏻 ✅ 



Cheers and Happy coding!👏👏👏

**Comment by mjeanbosco19** [comment]
> # Hello @mjeanbosco19 😃 🥳 🎉 🎈
> Great Job 😄 👏 You've done it. There is nothing else to say other than... it's time to merge it 👍🏻 ✅
> 
>  [ 68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f334b43326a44325163424f53632f67697068792e676966 ](https://camo.githubusercontent.com/7f4fd1ee2d3cc47007a12ab009e6dd3d4971c0240fd9476d7485954b937ae942/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f334b43326a44325163424f53632f67697068792e676966) [ ](https://camo.githubusercontent.com/7f4fd1ee2d3cc47007a12ab009e6dd3d4971c0240fd9476d7485954b937ae942/68747470733a2f2f6d65646961332e67697068792e636f6d2f6d656469612f334b43326a44325163424f53632f67697068792e676966)
> 
> Cheers and Happy coding!👏👏👏

Thanks @divinecharlotte

**Pull request merged**